src/ordering.jl

```diff
@@ -13,39 +13,13 @@ function <ₑ(a, b)
     if !istree(a) && !istree(b)
         T = typeof(a)
         S = typeof(b)
-        T===S ? isless(a, b) : nameof(T) < nameof(S)
+        return T===S ? isless(a, b) : nameof(T) < nameof(S)
     elseif istree(b) && !istree(a)
-        args = arguments(b)
-        if length(args) === 2
-            n1, n2 = !is_literal_number(args[1]) , !is_literal_number(args[2])
-            if n1 && n2
-                # both subterms are terms, so it's definitely firster
-                return true
-            elseif n1
-                return isequal(a, args[1]) || a <ₑ args[1]
-            elseif n2
-                return isequal(a, args[2]) || a <ₑ args[2]
-            else
-                # both arguments are not numbers
-                # This case when a <ₑ Term(^, [1,-1])
-                # so this term should go to the left.
-                return false
-            end
-        elseif length(args) === 1
-            # make sure a < sin(a) < b^2 < b
-            if isequal(a, args[1])
-                return true # e.g sin(a)*a should become a*sin(a)
-            else
-                return a<ₑargs[1]
-            end
-        else
-            # variables to the right
-            return false
-        end
+        return true
```

Comment:
Lol!! This was there so that you can get `a*sin(a)*b*b^2*sin(b)` I tried to more or less follow guide to the perplexed.

Reply:
It's very computational intensive to do that kind of ordering, since you have to traverse the whole tree to check it. SymEngine doesn't do it.

```julia
julia> using SymEngine

julia> @vars x y z
(x, y, z)

julia> sin(y) + x + x^2 + y^2 + z^2
x + x^2 + y^2 + z^2 + sin(y)
```

Reply:
True, but I'd like to keep a slow comparison that prints well. We can write a fast comparison for non-human purposes later.